recipes/cython_gsl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "CythonGSL" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/9c/62/838c21954ca98851f6d8b3778d45e1b1e8e7af98d6cca1dd951dca6b66ad/{{ name }}-{{ version }}.tar.gz
+  sha256: 177537590442be3ff0e21fb121cc3775a5dbf96611eeade317eee48d49282bd7
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - gsl
+  run:
+    - python
+    - gsl
+
+
+test:
+  requires:
+    - nose
+  commands:
+    - python setup_test.py build
+    - python setup_test.py install
+    - nosetest cython_gsl
```

Comment:
Then all the test stuff would be there after the main build and we could just run nosetests here without having the `python build install setup_test.py`.